### PR TITLE
15+ food items with images.

### LINKED ISSUE #9: Gather food dataset
15-30 food items. 
What to mine:
* Name
* Image
* General info about food. What it is.
* Safety level: good, bad, xxx
* In what quantities is it safe

Idea: use other reference apps and their lists and just steal data. Report what other information these app have about the foods.

### PR DESCRIPTION
**Pull request by kuredoro**
Closes #9

### CODE REVIEW

**Comment by Mexator** [comment]
Looks nice, we can merge, I guess.
And after you merge my branch, I will integrate with this dataset

File: data/data.json [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  "title": "Melon",
+  "category": "safe",
+  "details": "Melon possesses diuretic effect, removes fever. Berries are fed without seeds.",
+  },
+  {
+  "id": 17,
+  "title": "Banana",
+  "category": "with_care",
+  "details": "Bananas contains a lot of fibers, potassium and magenesium, vitamins B needed for healthy and beautiful fur, good appetite, quick growth and mental stability. The banana peels are preffered to be fed, since they contain more fibers. Too much of this fruit intails some risks: digestion disorder, overweigth, and risk of poisonning via chemicals used during banana's processing.",
+  },
```
Comment (Mexator) [inline data/data.json]: Please, no trailing commas. Not sure, If I can parse this with some ready-to go parser

**Review by Mexator — CHANGES_REQUESTED** [review]
Check comment above

File: data/data.json (hunk 1 above)
Comment (kuredoro) [reply data/data.json]: Fixed.

**Review by Mexator — APPROVED** [review]

**Pull request merged**